src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -139,9 +137,9 @@
     },
     "CHANGE_PWD": {
         "TITLE": "Modifier le mot de passe",
-        "CURRENT_PWD": "Mot de Passe Actuel",
-        "NEW_PWD": "Nouveau Mot de Passe",
-        "CONFIRM_PWD": "Confirmer le Mot de Passe",
+        "CURRENT_PWD": "Mot de passe actuel",
+        "NEW_PWD": "Nouveau mot de passe",
+        "CONFIRM_PWD": "Confirmer le mot de passe",
         "SAVE_SUCCESS": "Mot de passe utilisateur modifié avec succès.",
         "PASS_TIPS": "8-128 caractères long avec 1 majuscule, 1 minuscule et 1 chiffre"
```

Comment:
"8-128 caractères long" does not sound french, is it? Maybe suppress "long"?
And can you add "avec au moins" to be coherent with line 103?